libexec/bats

```diff
@@ -55,7 +60,7 @@ expand_path() {
 BATS_LIBEXEC="$(abs_dirname "$0")"
 export BATS_PREFIX="$(abs_dirname "$BATS_LIBEXEC")"
 export BATS_CWD="$(abs_dirname .)"
-export PATH="$BATS_LIBEXEC:$PATH"
+export PATH="$BATS_LIBEXEC:$(getconf PATH 2>/dev/null):$PATH"
```

Comment:
I'm not a fan of how this reorders the PATH. Typically, the PATH value at this point would be user-defined paths + system path, e.g.

```
$HOME/bin:/usr/local/bin:{system paths}
```

After this line, PATH will be:

```
$BATS_LIBEXEC:{system paths}:$HOME/bin:/usr/local/bin:{system paths}
```

Could we have `getconf` in the end? The thing I don't understand is that how on Solaris these `getconf`d paths aren't in PATH to start with?